gui/WorldView.cxx

```diff
@@ -810,3 +815,44 @@ void WorldView::updateScale()
     d->renderer->ResetCameraClippingRange();
   }
 }
+
+#ifdef VTKWEBGLEXPORTER
+//-----------------------------------------------------------------------------
+void WorldView::exportWebGLScene(QString const& path)
+{
+  QTE_D();
+
+
+  vtkNew<vtkWebGLExporter> exporter;
+
+  int width = d->renderWindow->GetScreenSize()[0];
+  int height = d->renderWindow->GetScreenSize()[1];
+
+  /** WARNING: This is a workaround. There's a bug in
```

Comment:
If it isn't merged yet, I would encourage mentioning the merge request. Do you have a link?